docs/management/index-patterns.asciidoc

```diff
@@ -1,100 +1,202 @@
+such as strings or boolean values. The field mapping also determines
+how you can use the field, such as whether it can be searched or aggregated.
+
+[role="screenshot"]
+image:management/index-patterns/images/new-index-pattern.png["Create index pattern"]
+
+[float]
+==== Format the display of common field types
+
+Whenever possible, {kib} uses the same field type for display as
+{es}. However, some field types that {es} supports are not available
+in {kib}. Using field formatters, you can manually change the field type in {kib} to display your data the way you prefer
+to see it, regardless of how it is stored in {es}.
+
+For example, if you store
+date values in {es}, you can use a {kib} field formatter to change the display to mm/dd/yyyy format.
+{kib} has field formatters for
+<<field-formatters-string, strings>>,
+<<field-formatters-date, dates>>,
+<<field-formatters-geopoint, geopoints>>,
+and <<field-formatters-numeric, numbers>>.
+
+A popularity counter keeps track of the fields you use most often.
+The top five most popular fields and their values are displayed in *Discover*.
+
+To edit the field format and popularity counter, use the *Edit* modal.
+
+[role="screenshot"]
+image:management/index-patterns/images/edit-field-format.png["Edit field format"]
+
+[float]
+==== Refresh the data fields
+
+To pick up newly-added fields,
+refresh (image:management/index-patterns/images/refresh-icon.png[Refresh icon]) the index fields list.
+This action also resets the {kib} popularity counters for the fields.
+
+[float]
+[[default-index-pattern]]
+=== Set the default index pattern
+
+The first index pattern you create is automatically designated as the default pattern,
+but you can set any index pattern as the default.  The default index pattern is automatically selected when you first open *Discover* or create a visualization from scratch.
+
+. In *Index patterns*, click the index pattern name.
+. Click the star icon (image:management/index-patterns/images/star.png[Star icon]).
+
+[float]
+[[delete-index-pattern]]
+=== Delete an index pattern
+
+This action removes the pattern from the list of saved objects in {kib}.
+You will not be able to recover field formatters, scripted fields, source filters,
+and field popularity data associated with the index pattern. Deleting an
+index pattern does not remove any indices or data documents from {es}.
+
+WARNING:  Deleting an index pattern breaks all visualizations, saved searches, and other saved objects that reference the pattern.
+
+. In *Index patterns*, click the index pattern name.
+. Click the delete icon (image:management/index-patterns/images/delete.png[Delete icon]).
+
+[float]
+=== What’s next
 
-To drill down into the fields and associated data types in an index pattern,
-click its name in the *Index patterns* overview page.
-For more information, refer to <<managing-fields, Index Patterns and Fields>>.
+* Learn about <<scripted-fields,scripted fields>> and how to create data on the fly.
```

Comment:
Are you adding more items to this list? If not, it should stand on it's own instead of an unordered list.